aspnetcore/blazor/hosting-models.md

```diff
@@ -257,13 +339,21 @@ Blazor WebAssembly, including Progressive Web Apps (PWAs), apps run using the .N
 
 :::moniker range="< aspnetcore-6.0"
 
+Blazor Server apps generally execute on the server quickly. However, Blazor Server apps are usually slower than other types of apps that execute natively on the client.
+
 Blazor WebAssembly apps run using the .NET runtime for WebAssembly, which is slower than running directly on the platform.
 
 :::moniker-end
 
 ### App code secure and private on the server
 
-:::moniker range=">= aspnetcore-6.0"
+:::moniker range=">= aspnetcore-8.0"
+
+Maintaining app code securely and privately on the server is a built-in feature of components rendered for the Blazor Server hosting model. Components rendered for the Blazor WebAssembly hosting model, static Blazor WebAssembly apps, and Blazor Hybrid apps can use server-based APIs to access functionality that must be kept private and secure. The considerations for developing and maintaining server-based APIs described in the [Direct access to server and network resources](#direct-access-to-server-and-network-resources) section apply. If the development and maintenance of server-based APIs isn't desirable for maintaining secure and private app code, render components for the Blazor Server hosting model.
```

Comment:
> Components rendered for the Blazor WebAssembly hosting model, static Blazor WebAssembly apps, and Blazor Hybrid apps

This phrase is pretty tedious and is used in several places. Can we change it to "Components rendered using the Blazor WebAssembly and Blazor Hybrid hosting models..."?